include/osgGA/FreeCameraManipulator

```diff
@@ -0,0 +1,64 @@
+#pragma once
+#ifndef __OSGGA_FREECAMERAMANIPULATOR__
+#define __OSGGA_FREECAMERAMANIPULATOR__
+
+#include <osgGA/GUIEventHandler>
+#include <osgGA/StandardManipulator>
+
+
+namespace osgGA
+{
+
+class FreeCameraManipulator : public osgGA::StandardManipulator
+{
+public:
+    FreeCameraManipulator();
+	FreeCameraManipulator(const FreeCameraManipulator& om, const osg::CopyOp& copyOp = osg::CopyOp::SHALLOW_COPY);
+
+    virtual osg::Matrixd getMatrix() const;
```

Comment:
There is no need to declare these as virtual since they are already declared as virtual in the base class.